from PyQt5 import QtCore, QtGui, QtWidgets


class Ui_AboutWindow(object):
    def setupUi(self, AboutWindow):
        AboutWindow.setObjectName("AboutWindow")
        AboutWindow.resize(403, 250)
        AboutWindow.setMaximumSize(QtCore.QSize(403, 250))
        icon = QtGui.QIcon.fromTheme(":/image/icon.png")
        AboutWindow.setWindowIcon(icon)
        AboutWindow.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.centralwidget = QtWidgets.QWidget(AboutWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.headingAbout = QtWidgets.QLabel(self.centralwidget)
        self.headingAbout.setGeometry(QtCore.QRect(170, 0, 51, 31))
        font = QtGui.QFont()
        font.setFamily("MS Sans Serif")
        font.setPointSize(12)
        font.setBold(True)
        font.setWeight(75)
        self.headingAbout.setFont(font)
        self.headingAbout.setStyleSheet("background-color: rgb(0, 170, 0);\n"
"color: rgb(0, 0, 0);")
        self.headingAbout.setObjectName("headingAbout")
        self.image = QtWidgets.QLabel(self.centralwidget)
        self.image.setGeometry(QtCore.QRect(30, 60, 121, 121))
        self.image.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.image.setText("")
        self.image.setPixmap(QtGui.QPixmap(":/image/icon.png"))
        self.image.setScaledContents(True)
        self.image.setObjectName("image")
        self.content = QtWidgets.QLabel(self.centralwidget)
        self.content.setGeometry(QtCore.QRect(200, 50, 181, 51))
        font = QtGui.QFont()
        font.setFamily("MS Sans Serif")
        font.setPointSize(10)
        self.content.setFont(font)
        self.content.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.content.setAlignment(QtCore.Qt.AlignLeading|QtCore.Qt.AlignLeft|QtCore.Qt.AlignTop)
        self.content.setWordWrap(True)
        self.content.setObjectName("content")
        self.versionNumber = QtWidgets.QLabel(self.centralwidget)
        self.versionNumber.setGeometry(QtCore.QRect(60, 190, 61, 20))
        self.versionNumber.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.versionNumber.setObjectName("versionNumber")
        self.nameApp = QtWidgets.QLabel(self.centralwidget)
        self.nameApp.setGeometry(QtCore.QRect(40, 30, 111, 31))
        font = QtGui.QFont()
        font.setFamily("MS Sans Serif")
        font.setPointSize(15)
        font.setBold(True)
        font.setWeight(75)
        self.nameApp.setFont(font)
        self.nameApp.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.nameApp.setObjectName("nameApp")
        self.label = QtWidgets.QLabel(self.centralwidget)
        self.label.setGeometry(QtCore.QRect(200, 110, 111, 16))
        self.label.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.label.setObjectName("label")
        self.label_2 = QtWidgets.QLabel(self.centralwidget)
        self.label_2.setGeometry(QtCore.QRect(240, 140, 61, 31))
        self.label_2.setStyleSheet("background-color: rgb(0, 170, 0);")
        self.label_2.setText("")
        self.label_2.setPixmap(QtGui.QPixmap(":/Logo/thunder_logo.png"))
        self.label_2.setScaledContents(True)
        self.label_2.setObjectName("label_2")
        AboutWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(AboutWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 403, 21))
        self.menubar.setObjectName("menubar")
        AboutWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(AboutWindow)
        self.statusbar.setObjectName("statusbar")
        AboutWindow.setStatusBar(self.statusbar)

        self.retranslateUi(AboutWindow)
        QtCore.QMetaObject.connectSlotsByName(AboutWindow)

    def retranslateUi(self, AboutWindow):
        _translate = QtCore.QCoreApplication.translate
        AboutWindow.setWindowTitle(_translate("AboutWindow", "About"))
        self.headingAbout.setText(_translate("AboutWindow", "About"))
        self.content.setText(_translate("AboutWindow", "PyQRCode is a Python application for Generating and Reading QRCode."))
        self.versionNumber.setText(_translate("AboutWindow", "Version: 1.0"))
        self.nameApp.setText(_translate("AboutWindow", "PyQRCode"))
        self.label.setText(_translate("AboutWindow", "Developed by Thunder"))
import icon_rc
import image_rc


if __name__ == "__main__":
    import sys
    app = QtWidgets.QApplication(sys.argv)
    AboutWindow = QtWidgets.QMainWindow()
    ui = Ui_AboutWindow()
    ui.setupUi(AboutWindow)
    AboutWindow.show()
    sys.exit(app.exec_())
